tests/bluetooth/tester/src/gap.c

```diff
@@ -494,15 +541,30 @@ static void start_directed_advertising(const uint8_t *data, uint16_t len)
 	const struct gap_start_directed_adv_cmd *cmd = (void *)data;
 	struct gap_start_directed_adv_rp rp;
 	struct bt_le_adv_param adv_param;
+	uint16_t options = sys_le16_to_cpu(cmd->options);
```

Comment:
```suggestion
	const uint16_t options = sys_le16_to_cpu(cmd->options);
```

Reply:
@sjanc Are you opposed to this change?

Reply:
it isn't too common to constify non-pointer variables,  but I can do that if it matters

Reply:
That's true. It's OK to leave as is, but I still see it as an improvement.